appveyor.yml

```diff
@@ -1,15 +1,20 @@
 environment:
   matrix:
+    - nodejs_version: "8.11.3"
```

Comment:
Since our docs say `8.11.3` is our minimum supported node.js version, and since there are newer versions of `8.11.x`, then we should continue running CI against it. We can *probably* trust that the Node maintainers respects semver, but this will actually become more useful later on. `8.11.3` ships with an older npm v5, and as we evolve our test dapps to test more realistically against a build of embark installed with npm, it will be good to know if using an older version of npm causes problems.